### DISCUSSION TITLE
Prisma environment variable

### DISCUSSION THREAD

**Discussion by shoaibsharif** [prisma/prisma#2289] [Q&A] [answered]
I undestand we have to define the enviroment variable in `prisma` folder or in terminal in order to work with @prisma/cli. What if I have a .env in root folder? Or what if I would have environment variable in production server? I have tried it but it is not working. I have checked in documentation, but it didn't mention about that.

**Comment by ryanking1809** [comment, marked as answer]
I've been using [dotenv-cli](https://github.com/entropitor/dotenv-cli) locally with .env files in the root folder. On production, I've just been using my hosting service's environment setup.

package.json
```json
"scripts": {
		"start": "node src/index.js",
		"debug": "export DEBUG=* && dotenv -e .env.development -- node src/index.js",
		"build": "prisma generate",
		"start-dev": "dotenv -e .env.development -- node src/index.js",
		"migrate-save-dev": "dotenv -e .env.development -- npx prisma migrate save --experimental",
		"migrate-up-dev": "dotenv -e .env.development -- npx prisma migrate up --experimental",
		"start-prod": "dotenv -e .env.production -- node src/index.js",
		"migrate-save-prod": "dotenv -e .env.production -- npx prisma migrate save --experimental",
		"migrate-up-prod": "dotenv -e .env.production -- npx prisma migrate up --experimental"
	},
```

Also looks like a built-in solution is being worked on https://github.com/prisma/prisma/issues/1487

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
